Dockerfile

```diff
@@ -8,7 +8,8 @@ RUN set -eux \
 		musl-dev \
 		openssl-dev \
 		python3 \
-		python3-dev
+		python3-dev \
+		bash
```

Comment:
@jomach  thanks for the contribution! Any specifics on why `bash` is required for the linting?

Reply:
For my use case I need to run some more scripts on Jenkins. So I start this image as a slave and run the linting as one of 3 steps. The other steps are bash scripts

Reply:
For my use case I need to run some more scripts on Jenkins. So I start this image as a slave and run the linting as one of 3 steps. The other steps are bash scripts

Reply:
@jomach this is the build image. Anything added here will not be present in the final image. `bash` should be added on line 32